src/plugins/index_pattern_editor/public/components/preview_panel/status_message/status_message.tsx

```diff
@@ -97,21 +126,21 @@ export const StatusMessage: React.FC<StatusMessageProps> = ({
     statusMessage = (
       <span>
         <FormattedMessage
-          id="indexPatternManagement.createIndexPattern.step.status.partialMatchLabel.partialMatchDetail"
-          defaultMessage="Your index pattern doesn't match any indices, but you have {strongIndices} which
+          id="indexPatternEditor.status.partialMatchLabel.partialMatchDetail"
+          defaultMessage="Your index pattern doesn't match any sources, but you have {strongIndices} which
```

Comment:
Does something like this work?

Your index pattern doesn't match any data streams, indices, or aliases, but some sources are similar.